clusterloader2/testing/load/config.yaml

```diff
@@ -270,7 +271,11 @@ steps:
       objectTemplatePath: networkpolicy.yaml
     {{end}}
     - basename: medium-deployment
+    {{if $ENABLE_DNSTESTS}}
+      objectTemplatePath: dns-deployment.yaml
```

Comment:
I'm fine with that conceptually, but I would like to clarify one thing - there are basically O(7.5*#nodes) pods in the cluster coming from this deployment during the test (so ~35000 in 5k-node cluster).

If we can handle that and this will work - I'm fine with that. I'm just ensuring that we will be able to handle that.

Reply:
Each pod will do 10 queries per second. We might run into dnsmasq max connections issue perhaps.  Is there a different config to change, so we have fewer pods to start with - like maybe 5k pods in a 5k node cluster?

Reply:
What is expected number of pods running dns-perf? Is it 5k? Less or more?

We can modify only some fraction of medium-deployments (so that we can control by variable a number of pods from 0 to 37k with steps of size 1500 pods).

Or if the desired number of pods is really small (so like resources on nodes are not a problem), we can simply run separate deployment (not a part of small,medium or big deployments).